pkg/types/types.go

```diff
@@ -103,6 +103,8 @@ const (
 // the rules.
```

Comment:
Can you also update the comments for the monitor inteface? It is not just for condition and event now.

Reply:
Good catch! Done.